### PR TITLE
small update

### PR DESCRIPTION
**Pull request by KingCosmic**
updated scale manager and added a resources page for commonly used links and such

### CODE REVIEW

File: RESOURCES.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+this is a list of resources useful in your gamedevelopment with phaser
```
Comment (16patsle) [inline RESOURCES.md]: Maybe create a heading?

**Review by 16patsle — CHANGES_REQUESTED** [review]

File: FAQ.md [hunk 2]
```diff
@@ -85,32 +85,11 @@ Right now this is just a collection of links that potentially may be useful:
 
 ## How do I scale a Phaser 3 game? Is there a ScaleManager in v3?
 
-_Last Updated: 2018-12-07_
```
Comment (jdotrjs) [inline FAQ.md:88]: Don't entirely drop the previous writeup; 3.16 comes with scale manager but it's worth leaving pointers to the examples in the event that folks are stuck < 3.16 for some reason.

File: RESOURCES.md (hunk 1 above)
Comment (jdotrjs) [reply RESOURCES.md]: > Maybe create a heading?

👍 

`# Resources`

File: RESOURCES.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+[Tile Extruder][te] this tool helps you extrude your tiles to solve the issue of tile bleeding.
```
Comment (jdotrjs) [inline RESOURCES.md]: Make this a list

**Review by jdotrjs — CHANGES_REQUESTED** [review]

**Comment by 16patsle** [comment]
@KingCosmic Planning on doing something here, or should we just let it rot until I find out I have time to do it myself?

**Comment by KingCosmic** [comment]
sorry I've been busy, had something that popped up earlier this month and will be off my plate this monday I can look into fixing it then if that's fine

**Comment by KingCosmic** [comment]
I fixed the requested changes, sorry for the lateness as I said had something pop up.

**Review by 16patsle — APPROVED** [review]

**Comment by 16patsle** [comment]
Thanks :smile:

**Review by jdotrjs — APPROVED** [review]

**Pull request merged**